client-admin/Dockerfile

```diff
@@ -5,11 +5,15 @@
 
 FROM docker.io/node:18-alpine
 
+# Set default NODE_ENV to production unless overridden at build time with --build-arg NODE_ENV=development
+ARG NODE_ENV
+ENV NODE_ENV ${NODE_ENV:-production}
```

Comment:
Since these client-specific Dockerfiles are going to be for development only (at least as things currently stand, since the prod building all happens in `file-server/Dockerfile`), this should maybe just be `development` by default?

On the other hand, it's annoying that we have separate Dockerfiles for each of the clients separately, and that all of this build logic is duplicated in the main `file-server/Dockerfile`. There were two reasons we decided to do that: 1) We wanted to support Podman as an alternative to Docker Desktop, and 2) It seemed like it might improve build processes on Heroku. We're already looking to move away from Heroku for deployment, and IIRC Podman may have been working on support for using `COPY FROM` from different Dockerfiles, which would let us reuse these images in the main `file-server/Dockerfile`. I'm also not sure how concerned we should be with podman specifically; Amazon is working on a similar thing to abstract away the need for Docker Desktop, and this may solve the problem more effectively. If we do go back to this way of doing things, the production default makes sense here.

Reply:
Looking now at https://github.com/compdemocracy/polis/pull/1135, and the [final comment ](https://github.com/compdemocracy/polis/pull/1135#issuecomment-1058338868) there from @willcohen pointed out that indeed it sounds like QEMU (which underlies podman support) may now support what we would need it to do to go back to having the separate Dockerfiles build the clients. So we should indeed consider going back to the old way of doing things.

With that in mind, it probably makes sense to leave `production` as the default.

Reply:
Agreed. Another thought that I considered is that file-server is technically optional and in some configurations these individual Dockerfiles are the source of truth and should be deployable without any additional help from file-server or docker compose or make.